From c9f586e1683be7c7808e376909bdb0f82311e293 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sat, 23 Dec 2023 15:54:37 +0200
Subject: [PATCH 013/155] ucm-imx93: spl: Update for 2023.04

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 board/compulab/ucm-imx93/spl.c | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/board/compulab/ucm-imx93/spl.c b/board/compulab/ucm-imx93/spl.c
index 8862606c390..1a0e0122e99 100644
--- a/board/compulab/ucm-imx93/spl.c
+++ b/board/compulab/ucm-imx93/spl.c
@@ -32,18 +32,38 @@
 #include <power/pmic.h>
 #include <power/pca9450.h>
 #include <asm/arch/trdc.h>
+#include <serial.h>
 #include "ddr/ddr.h"
 
 DECLARE_GLOBAL_DATA_PTR;
 
 int spl_board_boot_device(enum boot_device boot_dev_spl)
 {
+#ifdef CONFIG_SPL_BOOTROM_SUPPORT
 	return BOOT_DEVICE_BOOTROM;
+#else
+	switch (boot_dev_spl) {
+	case SD1_BOOT:
+	case MMC1_BOOT:
+		return BOOT_DEVICE_MMC1;
+	case SD2_BOOT:
+	case MMC2_BOOT:
+		return BOOT_DEVICE_MMC2;
+	default:
+		return BOOT_DEVICE_NONE;
+	}
+#endif
 }
 
 void spl_board_init(void)
 {
+	int ret;
+
 	puts("Normal Boot\n");
+
+	ret = ahab_start_rng();
+	if (ret)
+		printf("Fail to start RNG: %d\n", ret);
 }
 
 #if CONFIG_IS_ENABLED(DM_PMIC_PCA9450)
@@ -137,3 +157,9 @@ void board_init_f(ulong dummy)
 
 	board_init_r(NULL, 0);
 }
+
+#ifdef CONFIG_ANDROID_SUPPORT
+int board_get_emmc_id(void) {
+	return 0;
+}
+#endif
-- 
2.34.1

