From be8d04a7dad4723537b0f8244465a77b672bd808 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Mon, 18 Mar 2024 18:24:16 +0200
Subject: [PATCH 063/155] compulab: fdt: Update fdt_set_env_addr()

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 board/compulab/common/fdt.c | 16 +++++++++-------
 1 file changed, 9 insertions(+), 7 deletions(-)

diff --git a/board/compulab/common/fdt.c b/board/compulab/common/fdt.c
index bec94ad8097..4d95a207dac 100644
--- a/board/compulab/common/fdt.c
+++ b/board/compulab/common/fdt.c
@@ -58,14 +58,16 @@ int fdt_set_env_addr(void *blob)
 
 	fdt_setprop(blob, nodeoff, "env_off", tmp, sprintf(tmp, "0x%x", CONFIG_ENV_OFFSET));
 	fdt_setprop(blob, nodeoff, "env_size", tmp, sprintf(tmp, "0x%x", CONFIG_ENV_SIZE));
-	if(0 < env_dev) {
+
+	if (env_dev != -1) {
 		switch(env_part) {
-		case 1 ... 2:
-			fdt_setprop(blob, nodeoff, "env_dev", tmp, sprintf(tmp, "/dev/mmcblk%iboot%i", env_dev, env_part - 1));
-			break;
-		default:
-			fdt_setprop(blob, nodeoff, "env_dev", tmp, sprintf(tmp, "/dev/mmcblk%i", env_dev));
-			break;
+			case 2:
+			case 1:
+				fdt_setprop(blob, nodeoff, "env_dev", tmp, sprintf(tmp, "/dev/mmcblk%iboot%i", env_dev, env_part - 1));
+				break;
+			default:
+				fdt_setprop(blob, nodeoff, "env_dev", tmp, sprintf(tmp, "/dev/mmcblk%i", env_dev));
+				break;
 		}
 	}
 	return 0;
-- 
2.34.1

